internal/cloudapi/v2/v2_koji_test.go

```diff
@@ -444,8 +488,24 @@ func TestKojiCompose(t *testing.T) {
 		test.TestRoute(t, workerHandler, false, "PATCH", fmt.Sprintf("/api/worker/v1/jobs/%v", token), string(initJobResult), http.StatusOK,
 			fmt.Sprintf(`{"href":"/api/worker/v1/jobs/%v","id":"%v","kind":"UpdateJobResponse"}`, token, token))
 
-		// handle osbuild-koji #1
-		_, token, jobType, rawJob, _, err = workerServer.RequestJob(context.Background(), test_distro.TestArch3Name, []string{worker.WorkerJobTypeOSBuildKoji}, []string{""})
+		// Finishing of the goroutine handling the manifest job is not deterministic and as a result, we may get
```

Comment:
I kinda dislike using `workerServer.RequestJobById` because a real worker wouldn't use it. I think that dequeueing jobs and checking their IDs is a better way of differentiating.

Reply:
I reverted this change and jobs are now requested in the "usual" way.